start

```diff
@@ -502,4 +507,46 @@ popd () {
     command popd "$@" > /dev/null
 }
 
+function sorobanRpcStatus () {
+    horizonStatus
+    echo "waiting for soroban rpc to report ready state..."
+    COUNTER=1
+    while ! $(curl --silent --location --request POST 'http://localhost:8000/soroban/rpc' \
+                --header 'Content-Type: application/json' \
+        --data-raw '{
+          "jsonrpc": "2.0",
+          "id": 10235,
+          "method": "getHealth"
+          
+        }' | jq --exit-status '.result.status == "healthy"' 2>/dev/null | grep -o true || echo false ); 
+    do
+      if [ $COUNTER -gt 12 ]; then 
+          echo "waiting for soroban rpc to report ready state..."
+          COUNTER=1
+      else 
+          COUNTER=$[$COUNTER +1]    
+      fi 
```

Comment:
💡 I don't think we need to keep printing waiting. Just print it once should be sufficient. But I don't mind keeping this if this is preferrable UX for some reason.

Reply:
it helps indicate that process is progressing and not hung, so every minute it'll print a new line as a heartbeat kinda status that it's still ingesting, this will kick in during `--futurenet`

Reply:
I get that's how it works, but I don't think printing the same message out over and over is particularly useful. Unless it communicates some new information about progress, the user could just keep seeing this message in the event something is wrong, or not, it's hard to tell.

I think if we in parallel print out stellar-core's status, that will be more useful, because it will communicate more information about what is happening. The longest wait occurs because stellar-core is syncing.

Reply:
ok, loggoing ou core status info has been captured in new feature request - https://github.com/stellar/quickstart/issues/407